tasks/build.py

```diff
@@ -760,3 +760,65 @@ def check_build_permission(pr, event_info):
     else:
         log(f"{fn}(): GH account '{build_labeler}' is authorized to build")
         return True
+
+
+def request_bot_build_issue_comments(repo_name, pr_number):
+    status_table = {'arch': [], 'date': [], 'status': [], 'url': [], 'result': []}
+    cfg = config.read_config()
+ 
+    # for loop because github has max 100 items per request.
+    # if the pr has more than 100 comments we need to use per_page
+    # argument at the moment the for loop is for a max of 400 comments could bump this up
+    for x in range(1, 5):
+        curl_cmd = f'curl -L https://api.github.com/repos/{repo_name}/issues/{pr_number}/comments?per_page=100&page={x}'
+        curl_output, curl_error, curl_exit_code = run_cmd(curl_cmd, "fetch all comments")
+
+        comments = json.loads(curl_output)
+
+        for comment in comments:
+            # iterate through the comments to find the one where the status of the build was in
+            if config.read_config()["submitted_job_comments"]['initial_comment'][:20] in comment['body']:
+
+                # get archictecture from comment['body']
+                first_line = comment['body'][:comment['body'].find('/n')]
+                arch_map = get_architecture_targets(cfg)
+                for arch in arch_map.keys():
+                    target_arch = '/'.join(arch.split('/')[1:])
+                    if target_arch in first_line:
+                        status_table['arch'].append(target_arch)
```

Comment:
So this iterates over all defined architectures and adds it to `status_table['arch']` if it matches what it found in the first line of the comment. Alternatively, one could also take the value directly from comment?

Reply:
yes but than you first make it neccessary for 
``` 
initial_comment = New job on instance `{app_name}` for architecture `{arch_name}` for repository `{repo_id}` in job dir `{symlink}` 
``` 
to stay exactly the same now even if minor changes are made this will keep working.
Now I have tried to right this function in such a way that it allows changes to `initial_comment` in `app.cfg`